packages/cli/commands/project/watch.js

```diff
@@ -0,0 +1,268 @@
+const chokidar = require('chokidar');
+const path = require('path');
+const { default: PQueue } = require('p-queue');
+const {
+  addAccountOptions,
+  addConfigOptions,
+  setLogLevel,
+  getAccountId,
+  addUseEnvironmentOptions,
+} = require('../../lib/commonOpts');
+const { trackCommandUsage } = require('../../lib/usageTracking');
+const { logDebugInfo } = require('../../lib/debugInfo');
+const {
+  loadConfig,
+  validateConfig,
+  checkAndWarnGitInclusion,
+} = require('@hubspot/cli-lib');
+const {
+  logApiErrorInstance,
+  ApiErrorContext,
+} = require('@hubspot/cli-lib/errorHandlers');
+const { logger } = require('@hubspot/cli-lib/logger');
+const { isAllowedExtension } = require('@hubspot/cli-lib/path');
+const { shouldIgnoreFile } = require('@hubspot/cli-lib/ignoreRules');
+const {
+  provisionBuild,
+  uploadFileToBuild,
+  queueBuild,
+} = require('@hubspot/cli-lib/api/dfs');
+const { validateAccount } = require('../../lib/validation');
+const {
+  getProjectConfig,
+  validateProjectConfig,
+  pollBuildStatus,
+} = require('../../lib/projects');
+const { i18n } = require('@hubspot/cli-lib/lib/lang');
+
+const i18nKey = 'cli.commands.project.subcommands.watch';
+
+const loadAndValidateOptions = async options => {
```

Comment:
No need to define this. You should be able to import from [here](https://github.com/HubSpot/hubspot-cli/blob/bc84343fa5b2ab522a4bf3533141a0d3bbcaec25/packages/cli/lib/validation.js#L29)